### PR TITLE
Reversing orders sorting. Latest on top

### LINKED ISSUE #820: Default sorting the orders: Newest on top
Invert the current orders. New orders should go on top by default

Related: https://github.com/gnosis/dex-react/issues/822

### PR DESCRIPTION
**Pull request by alfetopito**
Closes #820

Last minute quick fix?

### CODE REVIEW

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr853--dexreact.review.gnosisdev.com

File: src/reducers-actions/orders.ts [hunk 1]
```diff
@@ -32,15 +32,31 @@ export const INITIAL_ORDERS_STATE = { orders: [], offset: 0 }
...
-    case 'OVERWRITE_ORDERS': {
       return { ...state, ...action.payload }
+    case 'OVERWRITE_ORDERS': {
+      const {
+        payload: { orders },
+      } = action
+
+      // default sorting order is ascending by order id
+      // since we are starting from 0, reverse to have the latest on top
+      orders.reverse()
```
Comment (Velenir) [inline src/reducers-actions/orders.ts:43]: Is it ok to mutate the array here? I guess it's not used anywhere else and only dispatched right after creation, but if it wasn't, mutating would be a bad idea.

Comment (W3stside) [reply src/reducers-actions/orders.ts:43]: YOUVE ACTIVATED MY TRAP CARD

Comment (anxolin) [reply src/reducers-actions/orders.ts:43]: I'll merge this for the user test, I think we can decide later what to do with this. My initial thought is that is fine, although if there's a risk, we can always create a new instance of the array before reversing.

If we do this, let's do it in another PR

Comment (alfetopito) [reply src/reducers-actions/orders.ts:43]: https://github.com/gnosis/dex-react/pull/870

**Pull request merged**